addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -157,4 +157,165 @@
 			<visible>!String.IsEmpty(PVR.ChannelNumberInput)</visible>
 		</control>
 	</include>
+	<include name="EpgGrid">
+		<definition>
+			<control type="epggrid" id="$PARAM[control_id]">
+				<left>0</left>
+				<top>0</top>
+				<right>20</right>
+				<bottom>340</bottom>
+				<orientation>$PARAM[control_orientation]</orientation>
+				<pagecontrol>60</pagecontrol>
+				<scrolltime tween="quadratic" easing="out">300</scrolltime>
+				<timeblocks>34</timeblocks>
+				<rulerunit>6</rulerunit>
+				<onleft>9000</onleft>
+				<onright>60</onright>
+				<onup>50</onup>
+				<ondown>50</ondown>
+				<viewtype label="$PARAM[viewtype_label]">list</viewtype>
+				<progresstexture border="$PARAM[progress_texture_border]" colordiffuse="button_focus">$PARAM[progress_texture]</progresstexture>
+				<rulerdatelayout width="1700" height="45" condition="$PARAM[has_rulerdate_layout]">
+					<control type="label">
+						<width>1700</width>
+						<height>45</height>
+						<font>font24</font>
+						<label>$INFO[ListItem.Label]</label>
+						<textcolor>button_focus</textcolor>
+						<align>center</align>
+						<aligny>center</aligny>
+					</control>
+				</rulerdatelayout>
+				<rulerlayout height="45" width="$PARAM[ruler_width]">
+					<control type="label">
+						<width>$PARAM[ruler_label_width]</width>
+						<height>45</height>
+						<font>font12</font>
+						<label>$INFO[ListItem.Label]</label>
+						<textoffsetx>10</textoffsetx>
+						<textcolor>button_focus</textcolor>
+					</control>
+				</rulerlayout>
+				<channellayout height="62" width="$PARAM[channel_width]">
+					<control type="label">
+						<left>2</left>
+						<top>-2</top>
+						<width>75</width>
+						<height>60</height>
+						<font>font32_title</font>
+						<label>$INFO[ListItem.ChannelNumberLabel]</label>
+						<align>center</align>
+						<aligny>center</aligny>
+					</control>
+					<control type="label" id="1">
+						<left>68</left>
+						<top>-2</top>
+						<width>$PARAM[channel_label_width]</width>
```

Comment:
Did not know this. Runtime-tested. Works like a charm without the 'width' definitions.